oozie/src/main/java/org/apache/falcon/oozie/feed/FeedReplicationWorkflowBuilder.java

```diff
@@ -145,6 +161,24 @@ private String getDefaultMapBandwidth() {
         return RuntimeProperties.get().getProperty("falcon.replication.workflow.mapbandwidth", "100");
     }
 
+    private String getDefaultAmMemory() {
```

Comment:
It would be better to move these method to OozieOrchestrationWorkflowBuilder and use it from there in FeedRetentionWorkflowBuilder and FeedReplicationWorkflowBuilder rather than doing same thing twice.